static/js/unread.js

```diff
@@ -648,10 +653,11 @@ export function mark_as_read(message_id) {
     // the following methods are cheap and work fine even if message_id
     // was never set to unread.
     unread_pm_counter.delete(message_id);
+    // Needs to called before updating `unread_topic_counter` below.
+    remove_message_from_unread_mention_topics(message_id);
```

Comment:
Can you check whether any of the other callers need either reordering or a comment adjustment?

Reply:
Doesn't seem like it.